File: modul_2/PRAK202-2510817220008-NamiraHumaira.py
a = float(input("Masukkan Nilai Pertama : "))
b = float(input("Masukkan Nilai Kedua : "))
hasil = a + b
if a.is_integer():
    print(f'Hasil dari penjumlahan nilai pertama "{int(a)}" dan nilai kedua "{b:.1f}" adalah "{hasil:.2f}"')
else:
    print(f'Hasil dari penjumlahan nilai pertama "{a:.2f}" dan nilai kedua "{b:.1f}" adalah "{hasil:.2f}"')

File: modul_2/PRAK202-2510817220008-NamiraHumaira.c
#include<stdio.h>
int main()
{
float pertama, kedua, hasil; 
printf("Masukkan nilai pertama : "); 
scanf("%f", &pertama); 
printf("masukkan nilai kedua : "); 
scanf("%f", &kedua);
hasil = pertama + kedua;
if (pertama == (int)pertama)
{
printf("Hasil dari penjumlahan nilai pertama \"%.0f\" dan nilai kedua \"%.1f\" adalah \"%.2f\" ", pertama, kedua, hasil);
}
else
{
printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua \"%.1f\" adalah \"%.2f\" ", pertama, kedua, hasil);
}
return 0;
}